samples: wifi: Add STA sample (#3)
D-Triveni/sdk-nrf: samples/wifi/sta/README.rst

.. _wifi_sta_sample:

> Wi-Fi: Sta

Reviewer: Station

###########

.. contents::
   :local:
   :depth: 2

> This sample allows Nordic Semiconductor's Wi-Fi chipset to connect with Access point.

Reviewer 2: Replace with "This sample demonstrates how to connect the Wi-Fi Station to a specified Access Point"

Requirements
************

The sample supports the following development kit:

.. table-from-sample-yaml::

Overview
********

> This sample can perform Wi-Fi operations "Connect and Disconnect" in the 2.4GHz and 5GHz bands depending on the capabilities supported in the underlying chipset.

Reviewer: our chipset supports both. So, may be `depending on capabilities of the access point` ?

> Using this sample, the development kit can associate with any Wi-Fi capable access point in :abbr:`STA (Station)` mode.

Reviewer 2: ```suggestion
Using this sample, the development kit can connect to the specified Access Point :abbr:`STA (Station)` mode.
```

Building and running
********************

.. |sample path| replace:: :file:`samples/wifi/sta`

.. include:: /includes/build_and_run_ns.txt

> Currently, the following configurations are supported:
> 
> * 7002 DK + QSPI

Reviewer 2: ```suggestion
Currently, the following board(s) are supported:

* nRF7002 DK
```

> 

Reviewer 2: Add details of how to configure the sample to connect to the desired Access Point - basically where to add the SSID, Password etc.

.. _wifi_station_sample:

Wi-Fi: Station
##############

.. contents::
   :local:
   :depth: 2

This sample demonstrates how to connect the Wi-Fi Station to a specified Access Point.

Requirements
************

The sample supports the following development kit:

.. table-from-sample-yaml::

Overview
********

This sample can perform Wi-Fi operations "Connect and Disconnect" in the 2.4GHz and 5GHz bands depending on the capabilities of an Access point.

Using this sample, the development kit can connect to the specified Access Point :abbr:`STA (Station)` mode.

Building and running
********************

.. |sample path| replace:: :file:`samples/wifi/sta`

.. include:: /includes/build_and_run_ns.txt

Currently, the following board(s) are supported:

* nRF7002 DK

The below parameters should be configured in ``prj.conf``:

>   Ssid

Reviewer: SSID, this is an acronym

Reviewer: Also, you can use a bulleted list or a table with "param - description - example"
